package.json

```
@@ -5,28 +5,32 @@
   "description": "simple starter",
   "main": "index.js",
   "scripts": {
-    "start": "npm run concurrent",
-    "concurrent": "concurrently --kill-others \"npm run dev:js\" \"npm run dev:css\" \"npm run dev:server\"",
-    "bundle": "concurrently \"rollup -c\" \"postcss app/index.css -o app/bundle.css\"",
     "dev:js": "rollup -c -w",
     "dev:css": "postcss app/index.css -o app/bundle.css -w",
-    "dev:server": "browser-sync start --server 'app' --files 'app/index.html,app/bundle.css,app/bundle.js' --no-open"
+    "dev:server": "browser-sync start --server 'app' --files 'app/index.html,app/bundle.css,app/bundle.js' --no-open",
+    "build:js": "rollup -c --environment NODE_ENV:production",
+    "build:css": "postcss app/index.css -o dist/bundle.css",
+    "build:html": "cp app/index.html dist/index.html",
+    "build": "run-s build:*",
```

this is a neat tool, the glob syntax is great! BUT, i've already got concurrently as a dep, this adds another dep that does the same thing (arguably more). we should pick one. 

also, whole reason concurrently was used was for concurrency!
```suggestion
    "build": "run-p build:*",
```

---

the only reason for the extra dep was because the `cp` command requires prior tasks to have created the target `dist` folder in advance. I’ll come up with a different way 👍

---

oh! hm, maybe this?

```suggestion
    "build": "mkdir dist && run-s build:*",
```

---

but if dist already exists... :scratches head:

---

Both `rollup` and `postcss` use `mkdirp`, so as long as they run first we're safe 😀